Lib/asyncio/base_events.py

```diff
@@ -601,29 +601,62 @@ def _check_running(self):
             raise RuntimeError(
                 'Cannot run the event loop while another loop is running')
 
-    def run_forever(self):
-        """Run until stop() is called."""
+    def run_forever_setup(self):
+        """Set up an event loop so that it is ready to start actively looping
+        to process events.
```

Comment:
By convention the first paragraph of a docstring must be a one-line summary.
```suggestion
        """Prepare for processing events.
```